ofj-template.tex

```diff
@@ -110,7 +110,7 @@
 
 
 %    \title[short text for running head]{full title}
-\title[\OF Journal publication]{\OF Journal publication}
+\title[\OF Journal Publication  - Use Title Case]{\OF Journal Publication - Use Title Case}
```

Comment:
This seems to indeed be the current format, so the change makes sense.

As this now starts the 2025 volume, it might however be a good time to question and potentially this style choice. In my eyes, it looks very wrong (especially with words that have specific capitalization, such as `preCICE`), and it is a practice that was invented as a workaround for limitations in typography that don't exist anymore. Many modern newspapers and websites have now switched to sentence case.

But this could also be part of a larger redesign of the style one could try at some point, to give the journal more distinct visual identity. In any case, let's discuss this in an issue or other context.

Reply:
I also agree with these relevant changes, which align with the current format. The editorial board decided to adopt title case because all members believe it looks better. However, any proposals for improvement are very welcome.